test/api/buffer/receivers/get/002-ok/request.json

```diff
@@ -0,0 +1,8 @@
+{
+  "path": "/v1/receivers/get-receiver",
+  "method": "GET",
```

Comment:
Mohol by sa adresar volat `002-get-ok`?
Chyba mi tu nejaky request na not found, `003-get-not-found`, vzdy by sme mali otestovat aj vsetky chyby co mozu nastat.